2023/YoctoSummit-2023.11/uefisb-yocto-part2.md

```diff
@@ -0,0 +1,227 @@
+  width="150px" style="margin-left:-20px">
+
+---
+
+# Agenda
+
+* `whoami`
+* Who we are?
+* Recap from first part
+  - booting secure embedded device
+  - UEFI Secure Boot
+  - meta-secure-core
+  - running image with UEFI Secure Boot integrated
+* Build improvements
+* UEFI Secure Boot compliance
+* Keys management
+* CI/CD integration
+* Demo
+
+---
+
+# `whoami`
+
+.center[<img
+  src="/remark-templates/3mdeb-presentation-template/images/tomasz_zyjewski.png"
+  width="150px">]
+
+.center[Tomasz Żyjewski]
+.center[_Embedded Systems Team Leader_]
+.right-column50[
+- over 4 years in 3mdeb
+- integration of update systems and OS creation for embedded devices
+- system security
+]
+.left-column50[
+- <a href="https://twitter.com/tomzy_0"><img
+  src="/remark-templates/3mdeb-presentation-template/images/twitter.png"
+  width="24px" style="margin-bottom:-5px; margin-left:-15px"/> @tomzy_0</a>
+- <a href="mailto:tomasz.zyjewski@3mdeb.com"><img
+  src="/remark-templates/3mdeb-presentation-template/images/email.png"
+  width="24px" style="margin-bottom:-5px; margin-left:-15px"/>
+  tomasz.zyjewski@3mdeb.com</a>
+]
+
+---
+
+# Who we are ?
+
+.center[.image-15[![](/remark-templates/3mdeb-presentation-template/images/coreboot-1024x1024.png)].image-15[![](/remark-templates/3mdeb-presentation-template/images/uefi-1024x1024.png)].image-15[![](/remark-templates/3mdeb-presentation-template/images/lvfs.png)].image-15[![](/remark-templates/3mdeb-presentation-template/images/yocto.png)]]
+.center[.image-35[![](/remark-templates/3mdeb-presentation-template/images/openpower.svg)]]
+
+- coreboot licensed service providers since 2016 and leadership participants
+- UEFI Adopters since 2018
+- Yocto Participants and Embedded Linux experts since 2019
+- Official consultants for Linux Foundation fwupd/LVFS project since 2020
+- IBM OpenPOWER Foundation members since 2020
+
+---
+
+# Recap from first part
+
+* Booting secure device
+  - Root of Trust (RoT): source that can always be trusted within a
+    cryptographic system
+  - Chain of Trust (CoT): ensuring that each subsequent element launched on the
+    device can be trusted, and thus verified by the previous element
+* Ensure that every stage of booting is verified
+* Let's focus on the process from powering up the platform to running the
+  operating system
+  - vary depending on the architecture we are dealing with
+  - in cases with UEFI Secure Boot we can talk about multistages boot chain
+    consist of BootROM, System ROM, Bootloader, OS
+
+---
+
+# Recap from first part
+
+* Explained in chapter 32 of UEFI Specification
+  - information from there should always take precedence over other documents
+* Key goal
+  - provide infrastructure for UEFI Image load time authentication
+  - UEFI Secure Boot authenticates OS Loaders and UEFI drivers
+  - Platform Owner manage platfrom's security policy and can check
+    check integrity and security of a given UEFI Image
```

Comment:
check check, grammarly needed

Reply:
Thanks, this one is fixed here https://github.com/3mdeb/conferences/pull/15/commits/836960d4d6218ad422a50583d6742ace442bcbc3#diff-d0591dad4f217325ddf75887a1f665e35e152ef737669037a1ac73cd94cd4d98R97 in the end I will run grammarly check once more